idaes/commands/config.py

```diff
@@ -1,137 +0,0 @@
-#################################################################################
-# The Institute for the Design of Advanced Energy Systems Integrated Platform
-# Framework (IDAES IP) was produced under the DOE Institute for the
-# Design of Advanced Energy Systems (IDAES).
-#
-# Copyright (c) 2018-2023 by the software owners: The Regents of the
-# University of California, through Lawrence Berkeley National Laboratory,
-# National Technology & Engineering Solutions of Sandia, LLC, Carnegie Mellon
-# University, West Virginia University Research Corporation, et al.
-# All rights reserved.  Please see the files COPYRIGHT.md and LICENSE.md
-# for full copyright and license information.
-#################################################################################
-"""Commandline Utilities for Managing the IDAES Config files"""
-# TODO: Missing docstrings
-# pylint: disable=missing-function-docstring
-
-# TODO: protected access issues
-# pylint: disable=protected-access
-
-__author__ = "John Eslick"
-
-import json
-import click
-import pyomo.common.config
-
-import idaes
-from idaes.commands import cb
-import idaes.config as cfg
-
-
-@cb.command(name="config-write", help="Write the IDAES config")
```

Comment:
Whilst removing this might be a good idea, in the spirit of backward compatibility we probably shouldn't just remove it without warning. If there is an alternative way to set the config then we should probably leave the options but have them return some sort of exception that tells users how to do this another way. Otherwise, we really should keep these but log a deprecation warning for the user.

Reply:
I hear you, and I can do that, but I'm about 100% sure that no one ever used this.

Reply:
On second thought, it would be better if possible to just remove this, since we don't want to configure the logger.  Alternatively we could just make all the commands still exist but print a message that they don't do anything anymore.

Reply:
That was my thought.